src/home/TabHome.tsx

```diff
@@ -74,13 +92,222 @@ function TabHome(_props: Props) {
+        })
+    } else {
+      addCKESBottomSheetRef.current?.snapToIndex(0)
+    }
+  }
+
+  function onPressSendMoney() {
+    AppAnalytics.track(TabHomeEvents.send_money)
+    !!cKESToken &&
+      navigate(Screens.SendSelectRecipient, {
+        defaultTokenIdOverride: cKESToken.tokenId,
+      })
+  }
+
+  function onPressRecieveMoney() {
+    AppAnalytics.track(TabHomeEvents.receive_money)
+    navigate(Screens.QRNavigator, {
+      screen: Screens.QRCode,
+    })
+  }
+
+  function onPressHoldUSD() {
+    AppAnalytics.track(TabHomeEvents.hold_usd)
+    !!cKESToken &&
+      !!cUSDToken &&
+      navigate(Screens.SwapScreenWithBack, {
+        fromTokenId: cKESToken.tokenId,
+        toTokenId: cUSDToken.tokenId,
+      })
+  }
+
+  function onPressWithdraw() {
+    AppAnalytics.track(TabHomeEvents.withdraw)
+    navigate(Screens.WithdrawSpend)
+  }
+
+  return (
+    <SafeAreaView testID="WalletHome" style={styles.container} edges={[]}>
+      <FlatCard testID="FlatCard/AddCKES" onPress={onPressAddCKES}>
+        <View style={styles.column}>
+          {!!cKESToken && (
+            <TokenIcon token={cKESToken} showNetworkIcon={false} size={IconSize.LARGE} />
+          )}
+          <Text style={styles.labelSemiBoldMedium}>{t('tabHome.addCKES')}</Text>
+        </View>
+      </FlatCard>
+      <View style={styles.row}>
+        <View style={styles.flex}>
+          <FlatCard testID="FlatCard/SendMoney" onPress={onPressSendMoney}>
+            <View style={styles.column}>
+              <Send />
+              <Text style={styles.labelSemiBoldMedium}>{t('tabHome.sendMoney')}</Text>
+            </View>
+          </FlatCard>
+        </View>
+        <View style={styles.flex}>
+          <FlatCard testID="FlatCard/RecieveMoney" onPress={onPressRecieveMoney}>
+            <View style={styles.column}>
+              <ArrowVertical />
+              <Text style={styles.labelSemiBoldMedium}>{t('tabHome.receiveMoney')}</Text>
+            </View>
+          </FlatCard>
+        </View>
+      </View>
+      <FlatCard testID="FlatCard/HoldUSD" onPress={onPressHoldUSD}>
+        <View style={styles.row}>
+          <Swap />
+          <View style={styles.flex}>
+            <Text style={styles.labelSemiBoldMedium}>{t('tabHome.holdUSD')}</Text>
+            <Text style={styles.bodySmallGray}>{t('tabHome.swapToUSD')}</Text>
+          </View>
+        </View>
+      </FlatCard>
+      <FlatCard testID="FlatCard/Withdraw" onPress={onPressWithdraw}>
+        <View style={styles.row}>
+          <Withdraw />
+          <Text style={styles.labelSemiBoldMedium}>{t('tabHome.withdraw')}</Text>
+        </View>
+      </FlatCard>
+      <AddCKESBottomSheet forwardedRef={addCKESBottomSheetRef} />
+    </SafeAreaView>
+  )
+}
+
+function FlatCard({
+  onPress,
+  testID,
+  ...props
+}: {
+  children: React.ReactNode
+  onPress: () => void
+  testID: string
+}) {
+  return <Touchable style={styles.flatCard} testID={testID} onPress={onPress} {...props} />
```

Comment:
have you tested this on android? we might need a borderRadius here so the touchable ripple displays correctly

Reply:
Thanks, you're right. fixed